x-pack/plugins/security_solution/public/management/pages/policy/view/ingest_manager_integration/endpoint_package_custom_extension/components/fleet_host_isolation_exceptions_card.tsx

```diff
@@ -0,0 +1,130 @@
+import {
+  PackageCustomExtensionComponentProps,
+  pagePathGetters,
+} from '../../../../../../../../../fleet/public';
+import {
+  GetExceptionSummaryResponse,
+  ListPageRouteState,
+} from '../../../../../../../../common/endpoint/types';
+import { useKibana, useToasts } from '../../../../../../../common/lib/kibana';
+import { useAppUrl } from '../../../../../../../common/lib/kibana/hooks';
+import { getHostIsolationExceptionsListPath } from '../../../../../../common/routing';
+import { getHostIsolationExceptionSummary } from '../../../../../host_isolation_exceptions/service';
+import { ExceptionItemsSummary } from './exception_items_summary';
+import { LinkWithIcon } from './link_with_icon';
+import { StyledEuiFlexGridGroup, StyledEuiFlexGridItem } from './styled_components';
+
+export const FleetHostIsolationExceptionsCard = memo<PackageCustomExtensionComponentProps>(
+  ({ pkgkey }) => {
+    const { getAppUrl } = useAppUrl();
+    const {
+      services: { http },
+    } = useKibana();
+    const toasts = useToasts();
+    const [stats, setStats] = useState<GetExceptionSummaryResponse | undefined>();
+    const hostIsolationExceptionsListUrlPath = getHostIsolationExceptionsListPath();
+    const isMounted = useRef<boolean>();
+
+    useEffect(() => {
+      isMounted.current = true;
+      const fetchStats = async () => {
+        try {
+          const summary = await getHostIsolationExceptionSummary(http);
+          if (isMounted.current) {
+            setStats(summary);
+          }
+        } catch (error) {
+          if (isMounted.current) {
+            toasts.addDanger(
+              i18n.translate(
+                'xpack.securitySolution.endpoint.fleetCustomExtension.hostIsolationExceptionsSummary.error',
+                {
+                  defaultMessage:
+                    'There was an error trying to fetch host isolation exceptions stats: "{error}"',
+                  values: { error },
+                }
+              )
+            );
+          }
+        }
+      };
+      fetchStats();
+      return () => {
+        isMounted.current = false;
+      };
+    }, [http, toasts]);
+
+    const hostIsolationExceptionsRouteState = useMemo<ListPageRouteState>(() => {
+      const fleetPackageCustomUrlPath = `#${
+        pagePathGetters.integration_details_custom({ pkgkey })[1]
+      }`;
+      return {
+        backButtonLabel: i18n.translate(
+          'xpack.securitySolution.endpoint.fleetCustomExtension.hostIsolationExceptionsSummary.backButtonLabel',
+          { defaultMessage: 'Back to Endpoint Integration' }
+        ),
+        onBackButtonNavigateTo: [
+          INTEGRATIONS_PLUGIN_ID,
+          {
+            path: fleetPackageCustomUrlPath,
+          },
+        ],
+        backButtonUrl: getAppUrl({
+          appId: INTEGRATIONS_PLUGIN_ID,
+          path: fleetPackageCustomUrlPath,
+        }),
+      };
+    }, [getAppUrl, pkgkey]);
+
+    return (
```

Comment:
I did not realize we did not have a common card component that each type of artifact could just use. This may be something worth doing in the future so that we don't keep creating unique cards per artifact type.

(cc/ @dasansol92 )

Reply:
Indeed, the reason why I did this so fast is because it was literally copy-paste work.